packages/runtime/test/integration/execute-transaction.ts

```diff
@@ -40,7 +39,7 @@ describe("Algorand Smart Contracts - Execute transaction", function () {
 
   function setupApp (): void {
     // create new app
-    appID = runtime.addApp({
+    runtime.addApp({
```

Comment:
could you please remove `fundAccount` from file below ?

Reply:
?

Reply:
If you look at the code we have one linter warning, can't comment on that file, but I can see it.

Reply:
ok, saw it. Check was running at that time, so i didn't see it either.

Reply:
no, checked there shouldn't be a warning here. `fundAccount` is being used in code
![image](https://user-images.githubusercontent.com/33264364/123984460-45b10f00-d9e2-11eb-898d-558b8d3e8b10.png)

but shows warning in github actions
![image](https://user-images.githubusercontent.com/33264364/123984313-27e3aa00-d9e2-11eb-9e25-aef36127a7f6.png)